cardano-node/src/Cardano/Tracing/Tracers.hs

```diff
@@ -231,20 +231,23 @@ instance ElidingTracer (WithSeverity (ChainDB.TraceEvent blk)) where
   doelide (WithSeverity _ (ChainDB.TraceAddBlockEvent (ChainDB.SwitchedToAFork{}))) = False
   doelide (WithSeverity _ (ChainDB.TraceAddBlockEvent (ChainDB.AddBlockValidation (ChainDB.InvalidBlock _ _)))) = False
   doelide (WithSeverity _ (ChainDB.TraceAddBlockEvent (ChainDB.AddBlockValidation ChainDB.CandidateContainsFutureBlocksExceedingClockSkew{}))) = False
-  doelide (WithSeverity _ (ChainDB.TraceAddBlockEvent (ChainDB.AddedToCurrentChain events _ _  _))) = null events
-  doelide (WithSeverity _ (ChainDB.TraceAddBlockEvent (ChainDB.PipeliningEvent{}))) = False
+  doelide (WithSeverity _ (ChainDB.TraceAddBlockEvent (ChainDB.AddBlockValidation _))) = True
+  doelide (WithSeverity _ (ChainDB.TraceAddBlockEvent (ChainDB.AddedToCurrentChain events _ _ _))) = null events
+  doelide (WithSeverity _ (ChainDB.TraceAddBlockEvent (ChainDB.PipeliningEvent{}))) = True
```

Comment:
> The addition of new trace messages made the eliding function inoparable. 'cardano.node.ChainDB:Notice' generates 1000s of log lines per second during sync.

I agree that suggests we should elide pipelining events.

The caveat is that we inspect benchmarking node logs in order to directly confirm pipelining is working as expected, and we won't be able to do that if eliding these events.

Relevant fact: the benchmarking runs (at least the ones we've used so far) don't involve bulk fetch/sync.

Thus, two possible ideas for follow-up:

  * disable pipelining or at least pipelining events during bulk fetch. (It's not entirely obvious how to set this switch, but it seems possible).
  
  * add a config flag we could set during benchmarking runs in order to disable elision of pipelining events

Reply:
> add a config flag

> I consider a simple fix

What about defining a small CPP flag that sets this to True for normal running, and False for benchmarking? That should be quite fast and small (although questionable, but it is small enough to be a "temporary fix" if this is going to be removed soon)